### PR TITLE
Show bot ratings on site

### PR DESCRIPTION
**Pull request by pseudomelon**
When drafting, a user can now see a button labelled 'Show Bot Ratings'. This button will display the ratings of various cards according to the chosen bot type. Pressing the button again will hide the ratings. The intent is to both give a novice user some drafting advice, as well as communicate the basic desires of a bot in a given state.

### CODE REVIEW

File: website/draft_site/drafts/templates/drafts/show_pack.html [hunk 1]
```diff
@@ -26,6 +29,13 @@
         <img src="{{ image_url }}" width="146" height="204" />
     {% endfor %}
 {% endif %}
+    <input type="button" onclick="unhide(this, 'bot_ratings') " value="Show Bot Ratings">
+
+    <div id="bot_ratings" class="hidden">
+        {% for rating in bot_ratings %}
+        <div >{{ rating.card.name }} : {{ rating.rating }}</div>
```
Comment (uponthesun) [inline website/draft_site/drafts/templates/drafts/show_pack.html]: Wouldn't it be better to show the entire Rating object for more info?

Comment (pseudomelon) [reply website/draft_site/drafts/templates/drafts/show_pack.html]: I had it this way at first, but the rating object is huge, and it obscures the thing you really want.

image
It's pretty difficult to read, imo.

**Comment by pseudomelon** [comment]
> 
> 
> Looks good overall! It could be better to store these values rather than generate them when viewing the draft, since the ratings would change for historical drafts if the tagging changes. Not sure yet though, so this is good for now.
> 
> The button position isn't quite right for the bot pages:
> 
> image



> 
> 
> Looks good overall! It could be better to store these values rather than generate them when viewing the draft, since the ratings would change for historical drafts if the tagging changes. Not sure yet though, so this is good for now.
> 
> The button position isn't quite right for the bot pages:
> 
> image

Not sure why I'm not allowed to respond to this comment directly. The button was fixed with a div. I went back and forth on the idea of storing the values and fetching them from the database for the user, but ultimately decided that since the button is only useful during an active draft, it's probably not important; if you can see the button and make decisions, it's because you're taking part in an active draft, so historical data doesn't make a lot of sense unless we're caching the bot networks (at which point we could just load the bot networks from SQL and recalculate the results anyways).

File: website/draft_site/drafts/templates/drafts/show_pack.html (hunk 1 above)
Comment (pseudomelon) [reply website/draft_site/drafts/templates/drafts/show_pack.html]: We could certainly put the data in tabular format, which would probably clean it up a fair bit. That would mean the tags would all occupy one column, though, which would not be great.

Comment (uponthesun) [reply website/draft_site/drafts/templates/drafts/show_pack.html]: The majority of the clutter comes from the full Card object being printed. You could do something like this to replace that with the card name: https://github.com/uponthesun/mtg-draft-ai/blob/master/mtg_draft_ai/brains.py#L152

Debatably we should make the name-only printing the default way to print out the object by changing/removing the __repr__ implementation on the Card object: https://github.com/uponthesun/mtg-draft-ai/blob/master/mtg_draft_ai/api.py#L34 I would be okay with that too.

**Comment by uponthesun** [comment]
I'd still like to get this in at some point. I think the only thing remaining is including the rest of the Rating info, with the card object replaced with the card name, which I think is worth doing since it'll help bot debugging a lot.

**Pull request merged**